content/news/core-monthly/2023/core-monthly-2023-04-01-2023-04-30.md

```diff
@@ -0,0 +1,353 @@
+---
+layout: post
+title:  "PrestaShop Core Monthly - April"
+subtitle: "An inside look at the PrestaShop codebase"
+date:   2023-05-09
+authors: [ kpodemski ]
+image: /assets/images/theme/banner-core-monthly.jpg
+twitter_image: /assets/images/theme/banner-core-monthly.jpg
+icon: icon-calendar
+tags:
+ - core-monthly
+---
+
+As we enter May, let's recap the key accomplishments within the PrestaShop project during April. We focused on vital security upgrades, namely PrestaShop 8.0.4 and 1.7.8.9, as well as putting the final touches on PrestaShop 8.1. A new project organization was also introduced, and the developer documentation received several updates. Read on to learn more about the latest developments in the PrestaShop project.
+
+![Core Monthly banner](/assets/images/theme/banner-core-monthly-wide.jpg)
+
+### New project organization
+
+On April 19th, 2023, [@eternoendless](https://github.com/eternoendless) unveiled a "A clearer, more transparent, more collaborative organization" for the project, that has been under development for the past few months. This revamped organization aims to streamline decision-making within the project. To learn more about the new structure, how it will function, and the next steps, we highly recommend reading the [dedicated article]({{< relref "/news/posts/2023/a-new-organization-for-the-open-source-project" >}}) on the Build blog. The organization was also presented during [April's Live Update](https://www.youtube.com/watch?v=SVbAQ__A6ns), you can watch the replay [here](https://youtu.be/SVbAQ__A6ns?t=179).
```

Comment:
```suggestion
On April 19th, 2023, [@eternoendless](https://github.com/eternoendless) unveiled "a clearer, more transparent, more collaborative organization" for the project, that has been under development for the past few months. This revamped organization aims to streamline decision-making within the project. To learn more about the new structure, how it will function, and the next steps, we highly recommend reading the [dedicated article]({{< relref "/news/posts/2023/a-new-organization-for-the-open-source-project" >}}) on the Build blog. The organization was also presented during [April's Live Update](https://www.youtube.com/watch?v=SVbAQ__A6ns), you can watch the replay [here](https://youtu.be/SVbAQ__A6ns?t=179).
```